packages/ast-tooling/index.ts

```diff
@@ -44,7 +48,7 @@ export function parseScript(content: string): AstTypes.Program {
         },
     }).program;
 
-    return jsAst;
+    return stripAst(jsAst, "loc");
```

Comment:
Can you explain why this was necessary? Is this just for better debug printing?

I was under the impression that `loc` should help the serialize to keep the code as close as possible to the original implementation that was parsed.

Reply:
The `loc` prop was needed to be stripped as it was causing false negatives during node comparisons, which often resulted in broken code when an adder is ran twice on a project (i.e. it would re-add a line that already existed). 

Consider the following case:
```js
const foo = {
	bar: 'baz'
};
```

```js
const foo = {
	bar: "baz"
};
```

these two cases are functionally equivalent, but to the node comparator, they are completely different as `loc` keeps track of which quote variation it uses.

Reply:
Sounds logical, thanks for the explanation!